apps/channels/tasks.py

```diff
@@ -72,9 +73,9 @@ def handle_facebook_message(self, team_slug: str, message_data: str):
 
 @shared_task(bind=True, base=TaskbadgerTask)
 def handle_turn_message(self, experiment_id: uuid, message_data: dict):
-    message = TurnWhatsappMessage.parse(message_data)
-    if not message.content_type == MESSAGE_TYPES.TEXT:
-        # TODO: Until we know what's up with the media links not showing up, we cannot continue
+    try:
+        message = TurnWhatsappMessage.parse(message_data)
+    except UnsupportedMessageTypeException:
```

Comment:
should these be logged at all?

Reply:
Unless we want to know when/if users are trying to send other types of messages, I think not, it will only clutter Sentry with stuff we can't fix. If we had a datadog integration, it would have been log this there